tools/src/test/scala/scala/scalanative/linker/ClassReachabilitySuite.scala

```diff
@@ -323,4 +324,32 @@ class ClassReachabilitySuite extends ReachabilitySuite {
     )
     (source, entry, reachable)
   }
+
+  /* The purpose of this test is to check that Test.main is NOT marked as `unreachable`.
+   * However linker would not either contain this method in its result since.
+   * Such scenario exists in issue #805, where inherited main method was not introduced in the scope
+   */
+  testReachable("inherited methods are reachable",
+                checkUnderApproximation = false) {
+    val source = """
+	  abstract class Parent {
+        def bar: Int
+		def main(): Unit = println(bar)
+      }
+      
+      object Test extends Parent {
+	    val bar: Int = 0
+      }
+	  """.stripMargin
```

Comment:
You have `stripMargin` here but no margin and strange indentation.

Reply:
Oh, thanks, forgot about that. I've removed margin since all other sources don't contain it, but forget to remove strip margin...and changed indentation later when testing...